src/alma-client.js

```diff
@@ -0,0 +1,16 @@
+const AlmaAxios = require('./api')
+const User = require('./user')
+
+class AlmaClient {
+  constructor (options = {}) {
+    this.api = new AlmaAxios({
+      region: options.region || 'eu',
+      key: options.apikey || process.env.ALMA_KEY,
```

Comment:
How does this handle neither of these being set?

Reply:
Do you mean `options.region` and `options.apikey`, or `options.apikey` and `process.env.ALMA_KEY`